tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -374,6 +377,26 @@ def _get_project_number(project_id, credentials):
     project = crm.projects().get(projectId=project_id).execute(num_retries=5)  # pylint: disable=no-member
     return project['projectNumber']
 
+# call to enable GCS bucket access level to Uniform access type
+def _enable_uniform_bucket_level_access(bucket_name):
+    """Enable uniform bucket-level access for a bucket"""
+
+    #commenting out API call as unable to access object iam_configuration 
+
+    #storage_client = storage.Client()
+    #bucket = storage_client.get_bucket(bucket_name)
+
+    #bucket.iam_configuration.uniform_bucket_level_access_enabled = True
+    #bucket.patch()
+
+    # using the gcloud sdk method to enable uniform bucket access, comment this out if API can be worked
```

Comment:
Can we use this?

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/storage/cloud-client/storage_enable_uniform_bucket_level_access.py#L23

Reply:
Yes , tried the same code example, but it does not recognize "iam_configuration" , hence had to switch to sdk

Reply:
What error are you facing? I tested iam_configuration.uniform_bucket_level_access_enabled = True on bucket instance and is working fine. Should use API methods instead of gsutil to avoid extra dependency.

Storage client version: 1.17.0